pkg/configuration/client.go

```diff
@@ -171,3 +176,30 @@ func (c *capsuleConfiguration) ForbiddenUserNodeAnnotations() *capsulev1beta1.Fo
 		Regex: c.retrievalFn().Annotations[capsulev1alpha1.ForbiddenNodeAnnotationsRegexpAnnotation],
 	}
 }
+
+func (c *capsuleConfiguration) IgnoredUserGroupsAnnotations() sets.String {
+	ignoredUserGroupSet := sets.NewString()
+	ignoredUserGroups, ok := c.retrievalFn().Annotations[capsulev1alpha1.IgnoredUserGroupsAnnotation]
+	if !ok {
+		return nil
```

Comment:
@prometherion we want to loose the `system:serviceaccounts:kube-system` service account. 

imho, if there is any specific request no accounts should be ignored, correct?

Reply:
We have to keep a default value here since we have to preserve backward compatibility. With the previous versions of Capsule, the hard-coded value was `system:serviceaccounts:kube-system`.

For your described use-case, we have to ignore any user, also the `system:serviceaccounts:kube-system`, and to achieve that, we just need to define the annotation key with an empty value.

With such strategy, we can cover the three cases:

1. default value when no annotation is in place
2. overridden value with the annotation
3. any value at all by defining the annotation with an empty value